ppr-ui/src/composables/address/factories/validation-factory.ts

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable */
+import { computed, ComputedRef, ref, Ref, toRefs, reactive } from '@vue/composition-api'
+import { maxLength, required } from 'vuelidate/lib/validators'
+
+import { AddressIF, SchemaIF } from '@/composables/address/interfaces'
+
+const useVuelidate = require("@vuelidate/core").default;
+
+// export const maxLength = (param: number) => helpers.withParams(
```

Comment:
I was going to leave that just in case vuelidates next update makes this work (this is what they show in their documentation to allow me access to the $params of a custom validator). As of now it doesn't work though so I will remove it haha